README.md

```diff
@@ -367,7 +362,7 @@ In short, all you need to do is to implement a
 a custom [ReporterProvider](ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/ReporterProvider.kt) using
 `META-INF/services/com.pinterest.ktlint.core.ReporterProvider`. Pack all of that into a JAR and you're done.
 
-To load a custom (3rd party) reporter use `ktlint --reporter=name,artifact=groupId:artifactId:version` / `ktlint --reporter=name,artifact=/path/to/custom-ktlint-reporter.jar`
+To load a custom (3rd party) reporter use `ktlint --reporter=name,artifact=/path/to/custom-ktlint-reporter.jar`
```

Comment:
Can you also add a note here about the old `groupId:artifactId:version` being scheduled for deprecation?

Reply:
Done, see https://github.com/pinterest/ktlint/pull/468/commits/b583aaf482fd4fb28b4d73467c34d87f143a3e13

Reply:
👍